tests/roots/test-ext-autodoc/target/typehints.py

```diff
@@ -88,3 +88,15 @@ def __init__(self, x: int) -> None:
 
         :param x: Some integer
         """
+
+
+class _ClassWithDocumentedInitAndStarArgs:
+    """Class docstring."""
+
+    def __init__(self, x: int, *args: int, **kwargs: int) -> None:
+        """Init docstring.
+
+        :param x: Some integer
+        :param *args: Some integer
+        :param **kwargs: Some integer
```

Comment:
Could you add a case for `args` and `kwargs` not having stars?

Reply:
I just pushed a commit with your proposal, the output is
```
  Class docstring.

   __init__(x, args, kwargs)

      Init docstring.

      Parameters:
         * **x** (*int*) -- Some integer

         * **args** (*int*) -- Some integer

         * **kwargs** (*int*) -- Some integer
```

which seems to be OK one to me, does this satisfy your first comment too?

Reply:
It seems the testcase you modified is configured as `autodoc_typehints_description_target='documented'`. It suppresses a bug. Could you check it again?

Reply:
Do you mean to add a similar test case but without `autodoc_typehints_description_target='documented'`?

Reply:
Yes. Please try this patch without `autodoc_typehints_description_target='documented'`. It will cause unexpected rendering. On my local, I saw both `args` and `*args` entries.

Reply:
I rebased and pushed 2 test cases without `autodoc_typehints_description_target='documented'`.

Reply:
Unfortunately, rebase was failed. It also brings commits on the master branch. So this PR is now broken. Could you check it please?

Reply:
I've just rebased onto 4.x, is it OK?